sdk/python/setup.py

```diff
@@ -65,6 +65,7 @@
     "fastapi>=0.68.0",
     "uvicorn[standard]>=0.14.0",
     "proto-plus<1.19.7",
+    "tensorflow-metadata>=1.0.0<2.0.0",
```

Comment:
I think you'll need a comma here between `>=1.0.0` and `<2.0.0`, right?

Reply:
you are correct, ty for seeing this. Totally slipped me.